src/SourceBuild/content/build.proj

```diff
@@ -26,7 +26,8 @@
     <ProjectReference Include="$(RepositoryEngineeringDir)finish-source-only.proj" Condition="'$(DotNetBuildSourceOnly)' == 'true'" />
   </ItemGroup>
 
-  <Target Name="PrintInfo">
+  <!-- Don't print this during restore. -->
+  <Target Name="PrintInfo" Condition="'$(MSBuildRestoreSessionId)' == ''">
```

Comment:
Weird, I didn't know this was possible. Does this still print info more than once?

Reply:
Why not just have it as a BeforeTargets="Build" (and remove the InitialTargets)

Reply:
I remember there was a reason back when we added the target but I don't see any reason to not do this with the current code. Thanks, updated.